Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -135,6 +134,41 @@ public void UpdateLayout()
 			Performance.Stop(reference);
 		}
 
+		protected int TabIndex { get; set; } = 0;
+
+		protected bool TabStop { get; set; } = true;
+
+		protected void UpdateTabStop() => TabStop = Element.IsTabStop;
+
+		protected void UpdateTabIndex() => TabIndex = Element.TabIndex;
+
+		public override AView FocusSearch(AView focused, [GeneratedEnum] FocusSearchDirection direction)
+		{
+			VisualElement element = Element as VisualElement;
+			int maxAttempts = 0;
+			var tabIndexes = element?.GetTabIndexesOnParentPage(out maxAttempts);
+			if (tabIndexes == null)
+				return null;
+
+			int tabIndex = element.TabIndex;
+			AView control = null;
+			int attempt = 0;
+			bool forwardDirection = !(
+				(direction & FocusSearchDirection.Backward) != 0 ||
+				(direction & FocusSearchDirection.Left) != 0 ||
+				(direction & FocusSearchDirection.Up) != 0);
+
+			do
+			{
+				element = element.FindNextElement(forwardDirection, tabIndexes, ref tabIndex);
+				var renderer = element.GetRenderer();
+				// use reflection to get the "Control" property from a specific renderer
+				control = renderer.GetType()?.GetProperty("Control")?.GetValue(renderer, null) as AView;
```

Comment:
You should be able to do something like this:

```
var renderer = element.GetRenderer() as ViewRenderer;
control = renderer?.Control;
```

and avoid having to use reflection. Same thing on iOS/macOS.

On Android, if the `ViewRenderer` cast comes back `null`, then it's probably a fast renderer and you can just try casting `renderer` to `AView` and use that result.